roles/ffh.mesh_batman/tasks/main.yml

```diff
@@ -6,8 +6,9 @@
 
 - name: Check if bat0 exists
   register: bat0_exists
-  stat:
-    path: /sys/class/net/bat0
+  cmd:
```

Comment:
for further reference, it should have been command,'cmd' ist the windows module...